pycomlink/processing/wet_dry/nearby_wetdry.py

```diff
@@ -15,9 +16,9 @@ def calc_distance_between_cml_endpoints(
     Calculating the distance from and to all start and endpoints of a network
     of CMLs using the Haversine distance formula. This includes the start and
     endpoint of each CML (which equals its length). The distance between start
-    and endpoint of a CML will be set to 0 for the case when r (the radius for
-    which CMLs are considered to be "nearby") is smaller than the actual length
-    of the CML
+    and endpoint of a CML will be set to 0. This is done for the case when r
```

Comment:
If I understand this correctly, it is better to write "This has to be done to correctly handle the case when r..."

Reply:
This sound better yes. Docstring is updated.